views/base.html

```diff
@@ -1,29 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-   <head>
-       {% include 'meta.html' %}
-      <title>SoA</title>
-      <link rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css">
-      <style>
-        @import url('https://fonts.googleapis.com/css?family=Lato');
-        html { font-family: 'Lato', sans-serif; }
-        {% if alerts %}
-          .alert {
-          padding: 5px;
-          border: 1px solid #2c8898;
-          border-radius: 5px;
-          cursor:pointer;
-          }
-          .info{
-          background-color: #b7dfea;
-          }
-          .success{
-          background-color: #85ffc7;
-          }
-          .danger{
-          background-color: #f4989c;
-          }
-          @media only screen and (min-width: 768px) { 
+    <head>
+        {% include 'meta.html' %}
+        <title>SoA</title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <link id="pagestyle" rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css" media="screen">
+        <!--
+            <script type="text/javascript">
+              function swapStyleSheet(sheet){
+                   document.getElementById('pagestyle').setAttribute('href',sheet);
+            
+              -->
+        <p>
```

Comment:
let's not include p tags inside head. If possible I'd like to move these options html to the accounts page. Perhaps a subheading of "This device"?